vars/sectionDeployToAKS.groovy

```diff
@@ -55,7 +57,7 @@ def call(params) {
 
   GithubAPI gitHubAPI = new GithubAPI(this)
   def testLabels = gitHubAPI.getLabelsbyPattern(env.BRANCH_NAME, 'enable_')
-  def depLabel = gitHubAPI.checkForDependenciesLabel(env.BRANCH_NAME)
+  def enableHelmLabel = gitHubAPI.checkForLabel(env.BRANCH_NAME, 'enable-helm')
```

Comment:
why are we not using the above code that retrieves all labels with `enable_` label, i.e. it should be `enable_helm` looking at the above code?

Reply:
good suggestion, will amend